### DISCUSSION TITLE
touch target not supported

### DISCUSSION THREAD

**Discussion by DennisLfromGA** [iptoux/kaliton#4] [Archive]
When I try to install the [touch](https://github.com/iptoux/kaliton/blob/main/targets/touch#L7) target the below error is thrown:

> "touch target is not supported on Debian."

-**DennisLfromGA**

**Comment by iptoux** [comment]
Yeah, some targets are not edited and need _changes/rework/remove_.

I don't know if the touch target is working in original crouton when you use debian as release, have you tested this? I will test this and response back later.

**Comment by DennisLfromGA** [comment]
@iptoux 

The [touch target logic](https://github.com/iptoux/kaliton/blob/main/targets/touch#L5) is below:
```
if [ "${TARGETNOINSTALL:-c}" = 'c' ] &&
        [ "$DISTRO" = 'debian' -o "$DISTRO" = 'kali' ]; then
    error 99 "touch target is not supported on Debian."
fi
```
So, crouton was throwing the error for both debain & kali.

I actually don't use touch that much so it's not really important to me.

-**DennisLfromGA**

**Comment by iptoux** [comment]
> @iptoux
> 
> The [touch target logic](https://github.com/iptoux/kaliton/blob/main/targets/touch#L5) is below:
> 
> ```
> if [ "${TARGETNOINSTALL:-c}" = 'c' ] &&
>         [ "$DISTRO" = 'debian' -o "$DISTRO" = 'kali' ]; then
>     error 99 "touch target is not supported on Debian."
> fi
> ```
> 
> So, crouton was throwing the error for both debain & kali.
> 
> I actually don't use touch that much so it's not really important to me.
> 
> -**DennisLfromGA**

Yep, i saw this too, so i think i close the issue now and label it as feature. My Chromebook doesn't support touch, so development and testing is not easy for me.

**Comment by DennisLfromGA** [comment]
OT but I also had a C64, along with a VIC-20 and a few Amigas <- they were *awesome!*

**Reply by iptoux** [reply]
You can contact me via [Linked.In](https://www.linkedin.com/in/maik-roland-damm/), please no OT in ideas.
